gaming/infrastructure/main.tf

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+variable "gcp_project" {
+  type = string
+}
+
+variable "spanner_config" {
+  type = object({
+    instance_name     = string
+    database_name     = string
+    configuration     = string
+    display_name      = string
+    processing_units  = number
```

Comment:
Is it intentional to only support fractional nodes and not whole (multiple) nodes?

Reply:
At this early stage, the main use of this sample will be code labs that people will run on their own environment. Processing units does support 1000 (1 node), which I figure is the most people will want, and most will be less. I also think this ties into the idea that fractional instances are viable even for production workloads that are still very small workloads.

That being said, nodes are definitely supported by the terraform module, so this can be modified later if there's a demand for it.